generated/nifake/nifake/_grpc_stub_interpreter.py

```diff
@@ -139,10 +145,7 @@ def fetch_waveform_into(self, number_of_samples):  # noqa: N802
         raise NotImplementedError('numpy-specific methods are not supported over gRPC')
 
     def function_with_repeated_capability_type(self, site_list):  # noqa: N802
-        self._invoke(
-            self._client.FunctionWithRepeatedCapabilityType,
-            grpc_types.FunctionWithRepeatedCapabilityTypeRequest(vi=self._vi, site_list=site_list),
-        )
+        raise NotImplementedError('function_with_repeated_capability_type is not supported over gRPC')
```

Comment:
Why are so many of these methods now raising NotImplementedError?

Reply:
These are updated from the metadata to match what the NI gRPC Device Server actually implements or not.